### PR TITLE
Tinkering with guns [Horribly broken]

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
Added Simultaneous to AN-94 and RM11B rifles, and changed the An-94 color to light_gray.

### CODE REVIEW

**Comment by Rivet-the-Zombie** [comment]
Praise be to the Abakan!

**Comment by BorkBorkGoesTheCode** [comment]
How do I fix this?

**Comment by mugling** [comment]
Run `make lint`

**Comment by BorkBorkGoesTheCode** [comment]
How, and what do I need to install?

**Comment by mugling** [comment]
If you are adding new fields do so in the same order to existing JSON. There are examples in `tools/format/example`. The linter is telling you which fields are in the wrong order.

File: data/json/items/gun/545x39.json [hunk 1]
```diff
@@ -35,18 +35,18 @@
...
     "material": [ "steel", "plastic" ],
     "symbol": "(",
-    "color": "brown",
+    "color": "dark_gray",
     "ammo": "545x39",
     "skill": "rifle",
     "dispersion": 150,
     "recoil": -10,
     "durability": 7,
-    "barrel_length": 1,
```
Comment (mugling) [inline data/json/items/gun/545x39.json]: Why did you swap this field around?

Comment (BorkBorkGoesTheCode) [inline data/json/items/gun/545x39.json]: Trying to figure out what the order is. Can't find any precedent.

Comment (mugling) [inline data/json/items/gun/545x39.json]: Look at the diff. You moved this field - it used to go before `valid_mod_locations`. The precedent are the files themselves (for linted content) or `tools/format/examples`.

Looking for this sort of thing is also useful in keeping the line count of your PR's low (irrespective of linting). Moving lines around unnecessarily also breaks git history.

File: data/json/items/gun/8x40mm.json [hunk 2]
```diff
@@ -43,7 +43,10 @@
     "dispersion": 10,
     "recoil": 125,
     "durability": 9,
-    "burst": 2,
+    "modes": [
```
Comment (mugling) [inline data/json/items/gun/8x40mm.json]: ...whereas here you make just an insertion (correct)

**Comment by mugling** [comment]
Separate concern - don't use `SIMULTANEOUS` - it implies multiple barrels and will eventually have different animation. Once the recoil PR (#18362) is done we could add another flag for fully delaying recoil until the end of the burst. You can leave this PR open with such a flag

**Comment by BorkBorkGoesTheCode** [comment]
Reverted

**Comment by mugling** [comment]
Don't let that bad rebase disuade you - it's time to learn `git` in all it's ugly glory!

**Comment by mugling** [comment]
Looks I forgot to actually add support for `SIMULTANEOUS` to the code so lets use that for this PR also and forget about the animation changes I proposed until the point when it's actually written (if ever)

**Comment by mugling** [comment]
Resubmit. You know what to do this time.

**Pull request closed without merging**